src/pages/images/ImageSelector.tsx

```diff
@@ -403,6 +407,12 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
               ]}
               value={type ?? ""}
             />
+            <CheckboxInput
+              aria-label="Show cached images"
+              checked={showCached}
+              label="Show cached images"
+              onChange={() => setShowCached((prev) => !prev)}
+            />
```

Comment:
Do we need this, maybe it can be always on and doesn't need user control.

Reply:
This was added after the discussion with @piperdeck . I think it could be useful to have the option to filter out cached images?

Reply:
this is now removed